### PR TITLE
feat(subscriptions): Create separate apps for content/subscription servers

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Hey @ianb, I read about your comment about preferring to create separate servers for the content and subscriptions, and when you did there was a lot of duplication.

I started on an attempt to create separate servers by extracting and sharing the bulk of the code and placing routes in server specific subdirectories. If you think this is a worthwhile approach, tomorrow (Thursday) I can try to update the tests and get it landed for you to use.

Note, I'm opening it as a PR against your PR because well.. it's easy... I'd open another PR against master.

### CODE REVIEW

File: packages/fxa-content-server/app/scripts/lib/config-loader.js [hunk 1]
```diff
@@ -31,15 +30,15 @@ ConfigLoader.prototype = {
     return this._readConfigFromHTML()
       .then(this._parseHTMLConfig)
       .then((config) => {
-        config.lang = $('html').attr('lang');
+        config.lang = document.querySelector('html').getAttribute('lang');
```
Comment (shane-tomlinson) [inline packages/fxa-content-server/app/scripts/lib/config-loader.js]: This is an orthogonal change to see if I could get rid of the jQuery dependency in the subscription page.

Comment (ianb) [reply packages/fxa-content-server/app/scripts/lib/config-loader.js]: I put this into #939

**Comment by lmorchard** [comment]
I think PR #943 is a satisfactory alternative to this and we can close.
